lib/rspec/support/object_formatter.rb

```diff
@@ -16,6 +16,10 @@ def self.default_instance
         @default_instance ||= new
       end
 
+      def self.default_max_formatted_output_length=(length)
+        default_instance.max_formatted_output_length = length
```

Comment:
I think it's fine to call this directly from expectations, both are private anyway.